imdb_django/management/commands/import_akas.py

```diff
@@ -0,0 +1,84 @@
+import csv
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.db.utils import IntegrityError  # Import IntegrityError
+
+from imdb_django.models import (
+    TitleAka,
+)  # Replace 'your_app_name' with your actual app name
+
+
+class Command(BaseCommand):
+    help = "Load data from TSV file into MySQL database"
+
+    def add_arguments(self, parser):
+        parser.add_argument("tsv_file", type=str, help="Path to the TSV file")
+
+    def handle(self, *args, **options):
+        tsv_file_path = options["tsv_file"]
+
+        with open(tsv_file_path, "r", encoding="utf-8") as tsv_file:
+            tsv_reader = csv.DictReader(tsv_file, delimiter="\t")
+
+            with transaction.atomic():  # Use a transaction for database consistency
+                row_count = 0  # Initialize a row count
+
+                for row in tsv_reader:
+                    title_id = row["titleId"]
+                    ordering = row["ordering"]
+                    title = row["title"]
+                    region = row["region"]
+                    language = row["language"]
+                    types = row["types"]
+                    attributes = row["attributes"]
+                    is_original_title = row["isOriginalTitle"] == "1"
+
+                    try:
+                        title_aka, created = TitleAka.objects.get_or_create(
+                            titleId_id=title_id,
+                            ordering=ordering,
+                            title=title,
+                            region=region,
+                            language=language,
+                            types=types,
+                            attributes=attributes,
+                            isOriginalTitle=is_original_title,
+                        )
```

Comment:
I think we should only check for title ID since it will be unique not the rest of the data. Please have a look at [get_or_create](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create) specially **defaults** param

Also see https://stackoverflow.com/a/1941246